<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageC</class>
 <widget class="QWidget" name="PageC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>730</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>751</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_searchRect">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>801</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic/all.qrc">:/pic/PageC-searchLine.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_destination">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 20px;
    color: #FFFFFF;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;  
    border-radius: 15px; /* 圆角 */  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-return-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-return-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_write_note">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>100</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;  
    border-radius: 15px; /* 圆角 */  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-note-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-note-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="selectType">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 未下拉时，QComboBox的样式 */
QComboBox {
    /*border: 1px solid gray;   /* 边框 */
    border-radius:15px;   /* 圆角 */
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */
    color: rgba(0,0,0,1);
    font: normal normal 16px &quot;Microsoft YaHei&quot;;
    background: transparent;
    text-align: AlignHCenter;
 
    color:rgb(255,255,255);/*字体颜色*/
    background-color: transparent;
   /* border-image: url(:/pic/PageB2-combo-bg2.png);/*背景图片，圆角*/
}
 
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid gray;   /* 选定项的虚框 */
    border-radius:10px;   /* 圆角 */
    padding-top:10px;    /*上下都留10px，是因为选中项是矩形，如果不留，移动到第一项和最后一项会变为直角，不好看*/
    padding-bottom:10px;
 
    color:rgb(160,160,160);
    border-image: url(:/pic/PageB2-combo-bg2.png);/*下拉框背景，圆角图片*/
}
 
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
 
    min-height: 50px;/*每项高度*/
}
 
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color:rgb(255,255,255);
}
 
/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color:rgb(255,255,255);
 
    border-image: url(:/pic/PageB2-combo-bg1.png);/*鼠标悬浮时背景图片*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
 	/*border-radius: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;*/

    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}

/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/pic/down.png);/*自定义图片填充*/
	width: 15px;/*设置该图标的宽高*/
	height: 15px;
}
</string>
   </property>
   <item>
    <property name="text">
     <string>全部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>商店</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>超市</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>便利店</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>书店</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>餐馆</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>食堂</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>咖啡厅</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>奶茶</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>卫生间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>休息室</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>图书馆</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_facilitiesCard">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>376</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic/all.qrc">:/pic/PageC-facilitie.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_go">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>51</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;  
    border-radius: 15px; /* 圆角 */  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-go-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-go-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-close-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-close-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_building">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>236</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 16px;
    color: #FFFFFF;
    background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>280</y>
     <width>311</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView {
	background-color: transparent;
	font: 7pt &quot;Microsoft YaHei&quot;;
	border: none;
	color : white
}

QListView::item {
    height: 60px;
}

QListView::item:hover {
	background-color: transparent;
	padding: 10px;
	border-left: 3px solid cyan;
}

QListView::item:selected {
	background-color: transparent;
	color: black;
	padding: 10px;
	border-left: 3px solid black;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="vehicleType">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>51</y>
     <width>151</width>
     <height>47</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* 未下拉时，QComboBox的样式 */
QComboBox {
    border: none;
    border-radius:22px;   /* 圆角 */
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */
    /*color: rgba(0,0,0,1);*/
    font: normal normal 20px &quot;Microsoft YaHei&quot;;
    background: transparent;
    text-align: AlignHCenter;
 
    color:rgb(255,255,255);/*字体颜色*/
    background-color: transparent;
    border-image: url(:/pic/PageB2-combo-bg1.png);/*背景图片，圆角*/
}
 
/* 下拉后，整个下拉窗体样式 */
QComboBox QAbstractItemView {
    outline: 0px solid gray;   /* 选定项的虚框 */
    border-radius:10px;   /* 圆角 */
    padding-top:10px;    /*上下都留10px，是因为选中项是矩形，如果不留，移动到第一项和最后一项会变为直角，不好看*/
    padding-bottom:10px;
 
    color:rgb(160,160,160);
    border-image: url(:/pic/PageB2-combo-bg2.png);/*下拉框背景，圆角图片*/
}
 
/* 下拉后，整个下拉窗体每项的样式 */
QComboBox QAbstractItemView::item {
 
    min-height: 47px;/*每项高度*/
}
 
/* 下拉后，整个下拉窗体越过每项的样式 */
QComboBox QAbstractItemView::item:hover {
    color:rgb(255,255,255);
}
 
/* 下拉后，整个下拉窗体被选择的每项的样式 */
QComboBox QAbstractItemView::item:selected {
    color:rgb(255,255,255);
 
    border-image: url(:/pic/PageB2-combo-bg1.png);/*鼠标悬浮时背景图片*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    width: 50px;/*设置按钮范围宽度*/
 	/*border-radius: 15px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;*/

    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
/*padding-right: 50px;*/
}

/*QCombobox右侧按钮的箭头图标*/
QComboBox::down-arrow {
	border-image: url(:/pic/down.png);/*自定义图片填充*/
	width: 15px;/*设置该图标的宽高*/
	height: 15px;
}

</string>
   </property>
   <item>
    <property name="text">
     <string>步行</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>骑行</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>电车</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_addto">
   <property name="geometry">
    <rect>
     <x>958</x>
     <y>600</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;  
    border-radius: 15px; /* 圆角 */  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-addto-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-addto-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setstart">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>600</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;  
    border-radius: 15px; /* 圆角 */  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-setstart-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-setstart-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gothere">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>600</y>
     <width>72</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {  
    border: none;  
    border-radius: 15px; /* 圆角 */  
    padding: 0px 0px 0px 10px; /* 上内边距、右内边距、下内边距、左内边距 */  
    color: rgb(255, 255, 255); /* 字体颜色 */  
    font: 15px &quot;Microsoft YaHei&quot;; /* 字体大小和名称 */  
    text-align: center;  
    background: transparent; /* 这里可以省略，因为后面会设置背景图片 */  
    background-image: url(:/pic/btn-gothere-normal.png); /* 设置背景图片 */  
	background-repeat:no-repeat;
}  
  
QPushButton:hover {  
    background-image: url(:/pic/btn-gothere-hover.png); /* 鼠标悬停时改变背景图片 */  
	background-repeat:no-repeat;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="legned">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>690</y>
     <width>227</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic/all.qrc">:/pic/legend.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="green">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>85</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic/all.qrc">:/pic/Infobox-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="orange">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>85</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic/all.qrc">:/pic/Infobox-orange.png</pixmap>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>409</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <zorder>label_bg</zorder>
  <zorder>label_searchRect</zorder>
  <zorder>label_destination</zorder>
  <zorder>btn_return</zorder>
  <zorder>btn_write_note</zorder>
  <zorder>label_facilitiesCard</zorder>
  <zorder>selectType</zorder>
  <zorder>btn_go</zorder>
  <zorder>btn_quit</zorder>
  <zorder>label_building</zorder>
  <zorder>listWidget</zorder>
  <zorder>vehicleType</zorder>
  <zorder>btn_addto</zorder>
  <zorder>btn_setstart</zorder>
  <zorder>btn_gothere</zorder>
  <zorder>legned</zorder>
  <zorder>green</zorder>
  <zorder>orange</zorder>
  <zorder>scrollArea</zorder>
 </widget>
 <resources>
  <include location="pic/all.qrc"/>
 </resources>
 <connections/>
</ui>
